ClientApp/viking-vault/src/Common/SideBar.tsx

```diff
@@ -2,21 +2,21 @@ import React from 'react'
 import './styles.css'
 import dashboard from '../Resources/images/dashboard.png'
 import arrow from '../Resources/images/arrowGRIdeschis.png'
+import placeholder from '../Resources/images/placeholder.png'
+import paymentMethod from '../Resources/images/payment-method.png'
+import block from '../Resources/images/password.png'
```

Comment:
Wouldn't really hurt to name imports related to the images be more descriptive, when I see the name `block` in the code I don't know what it refers to *Passive aggressive look*

Reply:
That was a bad import name, indeed, but we no longer need it, therefore it was deleted.